elodie/filesystem.py

```diff
@@ -10,11 +10,33 @@
 import re
 import shutil
 import time
+from os import path
+from configparser import RawConfigParser
 
 from elodie import geolocation
+from elodie import constants
 from elodie import log
 from elodie.localstorage import Db
 
+__DIR__ = None
+
+def get_dir():
+    global __DIR__
+    default_dir = "%Y-%m-%b"
+    if __DIR__ is not None:
+        return __DIR__
+
+    config_file = '%s/config.ini' % constants.application_directory
```

Comment:
Good catch. We should be using `RawConfigParser` everywhere. Insteaad of parsing it within this module I'd suggest adding a config module that can be imported anywhere we're parsing `config.ini` -- which appears to only be in `geolocation.py` at the moment.